nextcord/utils.py

```diff
@@ -851,6 +854,68 @@ def escape_mentions(text: str) -> str:
     return re.sub(r"@(everyone|here|[!&]?[0-9]{17,20})", "@\u200b\\1", text)
 
 
+def parse_raw_mentions(text: str) -> List[int]:
+    """A helper function that parses mentions from a string as an array of user IDs
+    matched with the syntax of ``<@user_id>``.
```

Comment:
```suggestion
    matched with the syntax of ``<@user_id>`` or ``<@!user_id>``.
```

Reply:
The nickname mention syntax is deprecated, but it is still supported like normal mentions, so I agree